documentation/ServicePrincipal/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "azure-sp",
```

Comment:
This is a very good sample to have. It shows how to create a service principal using the sdk. Some customers wanted this since they didn't want to use CLI/Powershell/Portal to do so.

We should make sure this sample is working with the new SDK.